src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "ibrahim-israr.webp"
+    "picture": "ibrahim-israr.webp",
+    "positions": { "S22": { "title": "Secretary", "tier": 3 } }
   },
   {
     "fullName": "Joel Anil John",
+    "picture": "joel-anil-john.webp",
     "positions": {
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "joel-anil-john.webp"
+      "S22": { "title": "Dev Officer", "tier": 19 },
+      "F22": { "title": "Algo Officer", "tier": 12 }
+    }
   },
   {
     "fullName": "Andy Lasso",
-    "positions": {
-      "F21": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "andy-lasso.webp"
+    "picture": "andy-lasso.webp",
+    "positions": { "F21": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Andrew Lau",
-    "positions": {
-      "S21": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "andrew-lau.webp"
+    "picture": "andrew-lau.webp",
+    "positions": { "S21": { "title": "Treasurer", "tier": 4 } }
   },
   {
     "fullName": "Minh Le",
-    "positions": {
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "minh-le.webp"
+    "picture": "minh-le.webp",
+    "positions": { "S22": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Nguyen Le",
-    "positions": {
-      "S22": {
-        "title": "Data Analyst",
-        "tier": 7
-      }
-    },
-    "picture": "nguyen-le.webp"
+    "picture": "nguyen-le.webp",
+    "positions": { "S22": { "title": "Data Analyst", "tier": 5 } }
   },
   {
     "fullName": "Tommy Le",
+    "picture": "tommy-le.webp",
     "positions": {
-      "F21": {
-        "title": "Treasurer",
-        "tier": 4
-      },
-      "S22": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "tommy-le.webp"
+      "F21": { "title": "Treasurer", "tier": 4 },
+      "S22": { "title": "Treasurer", "tier": 4 }
+    }
   },
   {
     "fullName": "Eugene Lee",
+    "picture": "eugene-lee.webp",
     "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      },
-      "F21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      },
-      "S22": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "eugene-lee.webp"
+      "S21": { "title": "nodebuds Officer", "tier": 22 },
+      "F21": { "title": "nodebuds Officer", "tier": 22 },
+      "S22": { "title": "nodebuds Officer", "tier": 22 }
+    }
   },
   {
     "fullName": "Nolan Lee",
-    "positions": {
-      "S22": {
-        "title": "Historian",
-        "tier": 12
-      }
-    },
-    "picture": "nolan-lee.webp"
+    "picture": "nolan-lee.webp",
+    "positions": { "S22": { "title": "Historian", "tier": 9 } }
```

Comment:
The updated "tier" for Historian is '9'

`"Historian": { "id": 10, "index": 600 },`

Reply:
false